packages/core/src/lib/server.js

```diff
@@ -36,30 +36,6 @@ const server = (patternlab) => {
             patternlab.config.paths.public.root
           )
         );
-        defaults.assets = [
```

Comment:
I know this is an old PR which is still WIP, but could you elaborate why this gets removed when the PR only is about ESLint changes?